### PR TITLE
refactor: Rename acronyms to camel case in SMS library 

### PR DESCRIPTION
**Pull request by JasperTimm**
This PR simply updates all references to the change in the SMS compression library where all acronyms are written as camel case instead of upper case.

This PR needs to wait on two things:
~~1. The reference in the compression library needs to use the dhis2 version, once this PR is merged: https://github.com/dhis2/sms-compression/pull/5~~
2. The previous PR with fixes should be merged first: https://github.com/dhis2/dhis2-core/pull/5159

### CODE REVIEW

File: dhis-2/dhis-services/dhis-service-core/pom.xml [hunk 1]
```diff
@@ -185,9 +185,9 @@
       <artifactId>commons-collections</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.github.dhis2</groupId>
+      <groupId>com.github.JasperTimm</groupId>
```
Comment (zubaira) [inline dhis-2/dhis-services/dhis-service-core/pom.xml:188]: @JasperTimm I have mered the PR for the sms-compression library. So I think we don't need to change this reference.

Comment (JasperTimm) [reply dhis-2/dhis-services/dhis-service-core/pom.xml]: @zubaira  - yep, updated to the dhis2 repo release now, thanks. Please keep in mind this PR includes the fix in #5159, so we should merge that first.

**Comment by larshelge** [comment]
#5159 is now merged @JasperTimm .

**Comment by larshelge** [comment]
@JasperTimm this PR looks good in general, but has a few conflicts now against master, would you be able to resolve?

**Pull request merged**